bcda/cli.go

```diff
@@ -67,18 +120,28 @@ func importCCLF8(fileMetadata cclfFileMetadata) error {
 	)
 
 	sc := bufio.NewScanner(file)
+	count := 0
+	validator := cclfFileValidator["CCLF8"]
+
 	for sc.Scan() {
 		b := sc.Bytes()
-		if len(bytes.TrimSpace(b)) > 0 {
+		bytelength := len(bytes.TrimSpace(b))
+		if bytelength > 0 && bytelength <= validator.maxRecordLength {
+			count++
+			// currently only errors if there are more records than we expect.
+			if count > validator.totalRecordCount {
+				return errors.New("maximum record count reached: did not match cclf0 validation")
```

Comment:
Can we log as much as possible here?  Filename being validated?  Expected count vs actual count?  Anything else?

Reply:
filename validation already happens earlier on. I can add extra logging for the expected vs. the actual though.

Reply:
If we fail validation here, I don't see anywhere indicating exactly which file failed.  We'll need transparency into knowing exactly the file that failed, so we can potentially request a re-send.

Reply:
gotcha! it's okay to log the filenames right?

Reply:
In an external Slack discussion with Eric:
```
The CCLF file names are okay for logging, since the chances of incorporating PII / PHI is low since it's internal CMS naming conventions
```